tests/api/patrons/test_patrons_rest.py

```diff
@@ -545,31 +545,51 @@ def test_patron_info(app, client, patron_martigny, librarian_martigny):
         url_for("api_patrons.info", access_token=no_scope_token.access_token)
     )
     assert res.status_code == 200
-    assert res.json == {"barcode": patron_martigny["patron"]["barcode"].pop()}
+    barcode = patron_martigny["patron"]["barcode"].pop()
+    assert res.json == {
+        "barcode": barcode,
+        "user_id": patron_martigny.user.id,
+        "patron_info": {"ORG1": {"patron_pid": patron_martigny.pid}},
+    }
 
     # full information with all scopes
     res = client.get(url_for("api_patrons.info", access_token=token.access_token))
     assert res.status_code == 200
     assert res.json == {
-        "barcode": "4098124352",
+        "barcode": barcode,
+        "user_id": patron_martigny.user.id,
         "birthdate": "1947-06-07",
         "fullname": "Roduit, Louis",
         "patron_types": [
             {
                 "expiration_date": patron_martigny["patron"]["expiration_date"]
                 + "T00:00:00",
-                "institution": "org1",
+                "institution": "ORG1",
+                "patron_pid": patron_martigny.pid,
                 "patron_type": "patron-code",
             }
         ],
+        "patron_info": {
+            "ORG1": {
+                "expiration_date": patron_martigny["patron"]["expiration_date"]
+                + "T00:00:00",
+                "institution": "ORG1",
```

Comment:
institution name or institution pid?

Reply:
Institution is the pid transformable with `RERO_ILS_APP_INSTITUTION_CODE_TRANSFORMATION`